lib/cartopy/io/img_tiles.py

```diff
@@ -881,16 +918,24 @@ def __init__(self, apikey, layer_id, api_version="v4",
             "About" tab for each layer displayed in the LINZ data service.
         api_version
             API version to use. Defaults to v4 for now.
-
+        desired_tile_form : str, optional
+            The desired format of the tile (defaults to "RGB").
+        cache : bool or pathlib.Path or str, optional
+            To allow offline use, as well as not spam tile providers, Cartopy
+            can create a local cache of previously fetched tiles. The default
+            path is ``cartopy.config["cache_dir"]``. If it is set to ``True``,
+            the default path is used. If it is set to a custom path, then this
+            path is used instead of the default one. If it is set to ``False``,
+            the tiles are downloaded each time.
         """
-        super().__init__(desired_tile_form=desired_tile_form, cache=cache)
+        super().__init__(layer=layer_id,
+                         desired_tile_form=desired_tile_form, cache=cache)
         self.apikey = apikey
-        self.layer_id = layer_id
```

Comment:
Same comment about backwards compatibility and mapping a `layer_id` property to `layer`.